app/src/main/java/com/standuptracker/RetrofitClientInstance.kt

```diff
@@ -6,7 +6,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 object RetrofitClientInstance {
 
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://api.npoint.io"
+    private const val BASE_URL = "https://api.npoint.io/"
```

Comment:
Since this URL isn't changing, it is best to create it as a const. Also, make sure your retrofit base url must end with a '/ ' or else there is a chance it won't work

Reply:
Careful with the / at the end.  This will be appended to the endpoint in the @GET() annotation, so you should consider both of them together.

Reply:
Yes, adding it here will break the url because we have a leading / in the @get() annotation

Reply:
Yea, that was my fault. I didn't realize that would happen.